jenkins/ci.suse.de/templates/cloud-mkcloud-job-upgrade-disruptive-template.yaml

```diff
@@ -20,7 +20,7 @@
             TESTHEAD=1
             cloudsource=develcloud{previous_version}
             upgrade_cloudsource=develcloud{version}
-            nodenumber=3
+            nodenumber=4
             cephvolumenumber=2
```

Comment:
I think so, but that can come as a revert to the original commit better than mix it in here I think